test/spanner.ts

```diff
@@ -1314,6 +1314,43 @@ describe('Spanner with mock server', () => {
           );
         });
 
+        it('should retry UNAVAILABLE during streaming with txn ID from inline begin response with parallel queries', async () => {
+          const err = {
+            message: 'Temporary unavailable',
+            code: grpc.status.UNAVAILABLE,
+            streamIndex: index,
+          } as MockError;
+          spannerMock.setExecutionTime(
+            spannerMock.executeStreamingSql,
+            SimulatedExecutionTime.ofError(err)
+          );
+          const database = newTestDatabase();
+
+          await database.runTransactionAsync(async tx => {
+            await Promise.all([tx!.run(selectSql), tx!.run(selectSql)]);
```

Comment:
Can we verify here that we get the expected (number of) rows?

Reply:
Done